device-registry/src/main/scala/org/genivi/sota/device_registry/GroupsResource.scala

```diff
@@ -75,20 +64,14 @@ class GroupsResource(namespaceExtractor: Directive1[AuthedNamespaceScope], devic
   def removeDeviceFromGroup(groupId: Uuid, deviceId: Uuid): Route =
     complete(db.run(GroupMemberRepository.removeGroupMember(groupId, deviceId)))
 
-  def getDiscardedAttrs(groupId: Uuid): Route =
-    complete(db.run(GroupInfoRepository.discardedAttrs(groupId)))
-
   val route: Route =
     (pathPrefix("device_groups") & namespaceExtractor) { ns =>
       val scope = Scopes.devices(ns)
-      (scope.post & path("from_attributes") & extractCreateGroupRequest) { request =>
-        createGroupFromDevices(request, ns)
-      } ~
       parameter('groupName.as[Name]) { groupName =>
-        (scope.post & path("group_info") & entity(as[Json])) { body =>
-          createGroupInfo(Uuid.generate(), groupName, ns, body)
+        (scope.post & path("group_info")) {
```

Comment:
Do we need both of these end-points? Or could we remove this one that mentions `"group_info"`?